python_modules/dagster/dagster/_core/execution/run_metrics_thread.py

```diff
@@ -241,7 +253,9 @@ def start_run_metrics_thread(
     container_metrics_enabled = _process_is_containerized()
     if not container_metrics_enabled and not python_metrics_enabled:
         if logger:
-            logger.debug("No collectable metrics, skipping run metrics thread")
+            logger.debug(
+                "No collectable metrics, skipping run metrics thread. Are we running in a container?"
```

Comment:
Can we make "Are we running in a container?" more clear / actionable, is this a directive to the user? Maybe just leave it out?

Reply:
how's this?